app/app.py

```diff
@@ -1,27 +1,49 @@
+import sys
+
 from controller.userController import UserController
 from util.decorators.tokenRequired import token_required
+from core.container import Container
```

Comment:
I'm not used to this, but I think this is awesome how you can just route to the directory file like a dot notation.